docs/env.md

```diff
@@ -80,6 +80,45 @@ console messages to enable, pass it to `makeConsole`. For example:
 - `DEBUG=SwingSet:vat` enable for user code, regardless of vat.
 - `DEBUG=SwingSet:vat,SwingSet:ls` enable for liveslots and user code, all vats
 
+## ENDO_DELIVERY_BREAKPOINTS
+
+The value of this option should be a JSON string identifying for which
+eventual-send message deliveries should a JS `debugger;` statement be executed.
+The format of the JSON file is
```

Comment:
Is the value a JSON string or a JSON file?
```suggestion
The value of this option should be a JSON string identifying for which
eventual-send message deliveries should a JS `debugger;` statement be executed.
The format of the JSON string is
```

Reply:
Done, thx